### PR TITLE
Release process

### PR DESCRIPTION
**Pull request by rowlando**
## What does this pull request do?

Documents the release process for this application for both non-production and production releases.

🔎 Nicer link to the new section: https://github.com/ministryofjustice/cla_backend/blob/release-process/README.md#releasing

screen shot 2018-09-12 at 16 42 26

## Any other changes that would benefit highlighting?

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+### Releasing to non-production
+
+1. Wait for [the Docker build to complete on CircleCI](https://circleci.com/gh/ministryofjustice/cla_backend) for the feature branch.
+1. Copy the `feature_branch.<sha>` reference from the `build` job's "Push Docker image" step. Eg:
+    ```
+    Pushing tag for rev [9a77ce2f0e8a] on {https://registry.service.dsd.io/v1/repositories/cla_backend/tags/dual-docker-registries.902c45d}
+    ```
+1. [Deploy `feature_branch.<sha>`](https://ci.service.dsd.io/job/DEPLOY-cla_backend/build?delay=0sec).
+    * `APP_BUILD_TAG` is the branch that needs to be released plus a specific 7-character prefix of the Git SHA. (dual-docker-registries.902c45d for the above example).
```
[the remark this answers is not in the document]
Comment (rowlando) [reply README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+See the list of users in `/admin/auth/user/`. Passwords are the same as the usernames.
+
+## Dev
+
+Each time you start a new terminal instance you will need to run the following commands to get the server running again:
+
+    source env/bin/activate
+    ./manage.py runserver 8000
+
+## Translaltions
```
[the remark this answers is not in the document]
Comment (rowlando) [reply README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+[![CircleCI](https://circleci.com/gh/ministryofjustice/cla_backend/tree/develop.svg?style=svg)](https://circleci.com/gh/ministryofjustice/cla_backend/tree/develop)
+    
+Backend API for the Civil Legal Aid Tool.
+
+## Dependencies
+
+-  `Virtualenv <http://www.virtualenv.org/en/latest/>`__
+-  `Most recent version of pip`
+-  `Python 2.7 <http://www.python.org/>`__ (Can be installed using ``brew``)
+-  `Postgres 9.3+ <http://www.postgresql.org/>`__
```
[the remark this answers is not in the document]
Comment (rowlando) [reply README.md]: 👍

File: README.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+When making changes to text (e.g. GraphML files) translations should be updated. To update translations run:
+
+     ./manage.py translations update
+
+
+## Scope Graphs
+
+* Edit the .graphml files, e.g. using a tool like [yEd](http://www.yworks.com/en/products/yfiles/yed/), to change the scope diagnosis trees
+* Run Django management command `python manage.py translations update` to update translations and templated graph files
```
[the remark this answers is not in the document]
Comment (rowlando) [reply README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    * `APP_BUILD_TAG` is the branch that needs to be released plus a specific 7-character prefix of the Git SHA. (`dual-docker-registries.902c45d` for the above example).
+    * `environment` is the target environment, select depending on your needs, eg. "demo", "staging", etc.
+    * `deploy_repo_branch` is the [deploy repo's](https://github.com/ministryofjustice/cla_backend-deploy) default branch name, usually master.
+
+### Releasing to production
+
+1. Please make sure you tested on a non-production environment before merging.
+1. Merge your feature branch pull request to `master`.
+1. Wait for [the Docker build to complete on CircleCI](https://circleci.com/gh/ministryofjustice/cla_backend/tree/master) for the `master` branch.
+1. Copy the `master.<sha>` reference from the `build` job's "Push Docker image" step. Eg:
```
Comment (sldblog) [inline README.md]: 📝 This repository still uses `develop` as the default branch, so we will have to change that after this PR.

**Pull request merged**